kivy/uix/actionbar.py

```diff
@@ -486,24 +486,39 @@ def on_use_separator(self, instance, value):
             group.use_separator = value
         self.overflow_group.use_separator = value
 
+    def remove_widget(self, widget):
+        super(ActionView, self).remove_widget(widget)
+        if isinstance(widget, ActionOverflow):
+            for item in widget.list_action_item:
+                self._list_action_items.remove(item)
+
+        if widget in self._list_action_items:
+            self._list_action_items.remove(widget)
+
     def _clear_all(self):
+        lst = []
+        for item in self._list_action_items:
+            lst.append(item)
         self.clear_widgets()
         for group in self._list_action_group:
             group.clear_widgets()
 
         self.overflow_group.clear_widgets()
         self.overflow_group.list_action_item = []
+        self._list_action_items = lst
```

Comment:
@KeyWeeUsr the function _clear_all is called from the three functions of layout when it has to layout the items in the resized window.Due to the clear_widget function all the items in the _list_action_items are removed resulting in nothing added to the list. (Just remove the line self._list_action_item = lst  and then run the code https://github.com/aerogelio/avposbug -->  add around 10 items using add button --> then maximize and minimize the window --> result is all items getting removed because _list_action_items was emptied by clear widgets ).